cognite/client/data_classes/filters.py

```diff
@@ -68,8 +68,20 @@ def _dump_property(property_: PropertyReference, camel_case: bool) -> list[str]
 class Filter(ABC):
     _filter_name: str
 
-    def dump(self, camel_case: bool = True) -> dict[str, Any]:
-        return {self._filter_name: self._filter_body(camel_case)}
+    def dump(self, camel_case_property: bool = False) -> dict[str, Any]:
```

Comment:
This is the bug introduced in v7, the default was changed from `False` to `True`